### ISSUE TITLE
Wifi APs not visible

### ISSUE THREAD

**Issue by all-solutions** [bigtreetech/PandaPWR#7]
I have bought two PWRs. They arrived today. Neither of them can find a Wifi AP. 
I have already done a factory reset (to be on the safe side) and reflashed the firmware. No change.

![image](https://github.com/user-attachments/assets/8e02f202-0a3b-4400-bd10-41f59d0ce1a3)

**Comment by dilixi** [comment]
> I have bought two PWRs. They arrived today. Neither of them can find a Wifi AP. I have already done a factory reset (to be on the safe side) and reflashed the firmware. No change.
> 
> ![image](https://private-user-images.githubusercontent.com/17761850/370220396-8e02f202-0a3b-4400-bd10-41f59d0ce1a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxODAzOTYsIm5iZiI6MTcyNzE4MDA5NiwicGF0aCI6Ii8xNzc2MTg1MC8zNzAyMjAzOTYtOGUwMmYyMDItMGEzYi00NDAwLWJkMTAtNDFmNTlkMGNlMWEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDEyMTQ1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkZTNkMGZmMGYxOTNhNmZjMjlkNGMzNTc5OGM5ZjZmYTRiYWU3NGFjNGZjMTY1MGRlZjU5MWNmMmI0NTFkODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.l9lzVRB8oNngTk1K0LylzH-8PH2rsf9ZO-gapvVwmqo)
Hi,please have a try:

1 Open a ap from your phone and see if there is a ap from the list.

2 Enter the ip(192.168.4.1) to your browser.

3 Click the F5 in your browser.

**Comment by dilixi** [comment]
> I have bought two PWRs. They arrived today. Neither of them can find a Wifi AP. I have already done a factory reset (to be on the safe side) and reflashed the firmware. No change.
> 
> ![image](https://private-user-images.githubusercontent.com/17761850/370220396-8e02f202-0a3b-4400-bd10-41f59d0ce1a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxODEwMjMsIm5iZiI6MTcyNzE4MDcyMywicGF0aCI6Ii8xNzc2MTg1MC8zNzAyMjAzOTYtOGUwMmYyMDItMGEzYi00NDAwLWJkMTAtNDFmNTlkMGNlMWEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDEyMjUyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2OWQ0MDRkZDhlOTMxODcwMTQ2MGJjNjk0YjAzMmY4Mjg5ZGU5MjBlODRhMDhmNzRkZmUyZGY5ZTZlZTY5NTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WjfTL5TzvI5o5ryCw1F0dBS0QE5wvVU5WDqda1IfYeI)

Could you mind upload a shotscreen with a picture which you have open the console from browser by clicking F12.

**Comment by dilixi** [comment]
> Great, that works. And I think the option of being able to manually type in the SSID name is great.
> 
> ![image](https://private-user-images.githubusercontent.com/17761850/370302683-96e4dc4a-242f-4310-922e-f022fd2d0a72.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyMzMzMjQsIm5iZiI6MTcyNzIzMzAyNCwicGF0aCI6Ii8xNzc2MTg1MC8zNzAzMDI2ODMtOTZlNGRjNGEtMjQyZi00MzEwLTkyMmUtZjAyMmZkMmQwYTcyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDAyNTcwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJiMjAxNjc5YmFhOTY2MzRkZWE3NzVhNmRhZDJlNGQ0MzEyM2RkNGIxNDNiZmZmMWQzMDg1ZmRlZWI0MzYwMGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QGKTg9MZirhmvtkgoU8wHBHoFZYZ-6ec4j3spLKQVKM)

Hi, can you help me reproduce this issue by updating this software, I need to fix this correctly in the next version.

https://github.com/dilixi/PandaPWR_Support/blob/main/panda_pwr_check_wifi_scan_failed.bin

**Comment by dilixi** [comment]
> > Hi, can you help me reproduce this issue by updating this software, I need to fix this correctly in the next version.
> 
> No problem, I'm pleased to do it. But unfortunately I'll have to wait until this afternoon before I can test it, the two printers are currently running.
> 
> I'll report back then...

Thank you.

**Comment by dilixi** [comment]
> ![image](https://private-user-images.githubusercontent.com/17761850/371046524-fcacf129-db09-4c88-8e7f-91f3c7b16ad1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczMzkwOTEsIm5iZiI6MTcyNzMzODc5MSwicGF0aCI6Ii8xNzc2MTg1MC8zNzEwNDY1MjQtZmNhY2YxMjktZGIwOS00Yzg4LThlN2YtOTFmM2M3YjE2YWQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA4MTk1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1YTE0NzAzOWM1YTNmMzc0YjQwY2NkNDMzZTgyOGZiOWIxMDMzOTU5YjU0ZTA0NmVkNTYxOGU1NjgwMzlkMzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GMOTchKZGewIKOPbguhViS_lagwMj8djyf6_T_Mz2Rc)

Thank you.
Are the surrounding APs set to hidden?

**Comment by dilixi** [comment]
> No, all visible. Also the AP of the second PWR is visible.
> 
> ![image](https://private-user-images.githubusercontent.com/17761850/371050104-2515f7ef-aa8b-4297-9fc4-12032b851ef1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNDA1MDksIm5iZiI6MTcyNzM0MDIwOSwicGF0aCI6Ii8xNzc2MTg1MC8zNzEwNTAxMDQtMjUxNWY3ZWYtYWE4Yi00Mjk3LTlmYzQtMTIwMzJiODUxZWYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA4NDMyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkOTAzNzdkNmY1ZGU2ZjkxMzAzMmE3YThkMzc2NTNlNWQzNDczNWVjOGE1N2I3NmE3ZjA0YjM4Y2FlMmY2YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HJtLJ1kt1tiSwPHxMAMlJX6qLOZw7BpXL-SH0HKS7jE)

Can you reproduce it 100%? Has it never worked?

**Comment by dilixi** [comment]
> > No, all visible. Also the AP of the second PWR is visible.
> > ![image](https://private-user-images.githubusercontent.com/17761850/371050104-2515f7ef-aa8b-4297-9fc4-12032b851ef1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNDA1MDksIm5iZiI6MTcyNzM0MDIwOSwicGF0aCI6Ii8xNzc2MTg1MC8zNzEwNTAxMDQtMjUxNWY3ZWYtYWE4Yi00Mjk3LTlmYzQtMTIwMzJiODUxZWYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA4NDMyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkOTAzNzdkNmY1ZGU2ZjkxMzAzMmE3YThkMzc2NTNlNWQzNDczNWVjOGE1N2I3NmE3ZjA0YjM4Y2FlMmY2YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HJtLJ1kt1tiSwPHxMAMlJX6qLOZw7BpXL-SH0HKS7jE)
> 
> Can you reproduce it 100%? Does it give a pop error message from web ui everytime? Can it succeed one time?
If you test it 10 times , can it have 1 time succeed?

**Comment by dilixi** [comment]
> ![image](https://private-user-images.githubusercontent.com/17761850/371046524-fcacf129-db09-4c88-8e7f-91f3c7b16ad1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNDI3MzIsIm5iZiI6MTcyNzM0MjQzMiwicGF0aCI6Ii8xNzc2MTg1MC8zNzEwNDY1MjQtZmNhY2YxMjktZGIwOS00Yzg4LThlN2YtOTFmM2M3YjE2YWQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA5MjAzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExNDQwY2MyOTY0NDlmN2MwNmE0MzdiNDFhYzQxNTJjMDRhOTIwNTJiMWU1MjhhZTg2YzhhZDZlYTA2MjZhNWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.iUl3TJMEdPhkFAR0BDmQPnF5r_DueJ_r3NdlwBksEwc)

Hi all-solutions，
I can not reproduce this issue, And I have changed the timeout from 5 seconds to 15 seconds, please help me to have a try again.

https://github.com/dilixi/PandaPWR_Support/blob/main/panda_pwr_scan_add_timeout.bin

Thank you.

**Comment by dilixi** [comment]
> Great, that works. And I think the option of being able to manually type in the SSID name is great.
> 
> ![image](https://private-user-images.githubusercontent.com/17761850/370302683-96e4dc4a-242f-4310-922e-f022fd2d0a72.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNDM2NjQsIm5iZiI6MTcyNzM0MzM2NCwicGF0aCI6Ii8xNzc2MTg1MC8zNzAzMDI2ODMtOTZlNGRjNGEtMjQyZi00MzEwLTkyMmUtZjAyMmZkMmQwYTcyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA5MzYwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFhOTJmZmUzYzQyMjMyMWFkODczN2Q2MTg0OTc0ZmQ1YWM1ZGFjZGViN2I5NDQyZWM5YTI2N2M2ZTA3MWYzZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-CpbeKOS8x-sQ98WH_fljLLw42F5tkZnMz1Yk9TjeHw)
 
 I have changed the scan type from PASSIVE to ACTIVE, but it need more time to wait the scan task.

**Comment by dilixi** [comment]
> panda_pwr_change_wifi_scan.bin

Thank you.
Could you please have a try with this one?
https://github.com/dilixi/PandaPWR_Support/blob/main/panda_pwr_active_with_errors.bin

**Comment by dilixi** [comment]
> I Think now it's working, need about 15 sec. for the AP scan
> 
>  PWR_4.mp4

Thank you very much.
I have change the min scan time to 100ms.

**Comment by AnalyticETH** [comment]
I am seeing this same issue. What firmware file should I use now?

**Comment by dilixi** [comment]
> I am seeing this same issue. What firmware file should I use now?

Hello,
Please try this:
https://github.com/dilixi/PandaPWR_Support/raw/refs/heads/main/panda_pwr_passive_100_150.bin

If nothing changes, you can use this, but the scan will take a long time.
https://github.com/dilixi/PandaPWR_Support/raw/refs/heads/main/panda_pwr_scan_add_retry.bin

Thank you.

**Comment by dilixi** [comment]
> I Think now it's working, need about 15 sec. for the AP scan
> 
>  PWR_4.mp4

@all-solutions 
Hello,
Do you have time to do more testing?
I'm not sure if this can take less time waiting for WiFi scanning.

https://github.com/dilixi/PandaPWR_Support/raw/refs/heads/main/panda_pwr_scan_quickly.bin

thanks.

**Comment by dilixi** [comment]
> Hi, yes, I can do that. But unfortunately only after Friday. I'm on a business trip at the moment.

Hi all-solutions,
If you place PandaPWR in an area with less WIFI network, it may be better, perhaps because the network channel is too crowded.

**Comment by AnalyticETH** [comment]
Quick update - I connected the PWR to the Panda Touch and it connected to my WiFi. I did not need to flash new firmware.

**Comment by dilixi** [comment]
> Quick update - I connected the PWR to the Panda Touch and it connected to my WiFi. I did not need to flash new firmware.

Hello, 
Thanks for your replay,but the fact is that very few people experience this problem and it can be fixed by flashing the new firmware.

**Comment by dilixi** [comment]
Hi all-solutions,
Do you have time to test this new firmware?

**Comment by dilixi** [comment]
> > Hi all-solutions, Do you have time to test this new firmware?
> 
> Hi dilixi, Yes, I can test it tomorrow afternoon.

Thanks.

**Comment by dilixi** [comment]
> Which version should I test? I'll be home in about 2 hours.

Hi,
This one,please.
https://github.com/bigtreetech/PandaPWR/blob/master/Firmware/1.0.0/panda_pwr-v1.0.0.bin

**Comment by dilixi** [comment]
> Works good. Took about 2-3 seconds for the wifi scan and all wifi networks are visible and selectable. I had previously made a factory reset to prevent the ESP from still having stored values somewhere. ![image](https://private-user-images.githubusercontent.com/17761850/380200204-010f4efe-9114-4b0a-8f5a-fd886b7b2cf0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4NjczNDAsIm5iZiI6MTcyOTg2NzA0MCwicGF0aCI6Ii8xNzc2MTg1MC8zODAyMDAyMDQtMDEwZjRlZmUtOTExNC00YjBhLThmNWEtZmQ4ODZiN2IyY2YwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDE0MzcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0ZGNiYzQwOWM3ODgzZGY1ZWFlNDY2YjA1ZjQzMTgxNmUxZTVkMzliNDM2MzgyNmI5ZDJlOTAwMDdlMDM4ZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VUBSlRXk7aSLzglAnoS72cyqdQcKfNXNGtEMMsragt8)

Good job.
Thank you very much.
Now I can close this issue.
